pod/import_video/templates/import_video/delete.html

```diff
@@ -0,0 +1,76 @@
+{% extends 'base.html' %}
+{% load i18n %}
+{% load static %}
+
+
+{% block breadcrumbs %}
+  {{ block.super }}
+  <li class="breadcrumb-item"><a href="{% url 'meeting:my_meetings' %}">{% trans 'My meetings' %}</a></li>
+  <li class="breadcrumb-item"><a href="{% url 'meeting:edit' meeting.meeting_id %}">{{meeting.name}}</a></li>
+  <li class="breadcrumb-item active" aria-current="page">{% trans "Delete" %}</li>
+{% endblock %}
+
+{% block page_title %}
+  {% blocktrans with name=meeting.name %}Delete the meeting <b>{{name}}</b>{% endblocktrans %}
```

Comment:
Est-ce que ce devrait etre "Delete the recording" ?

Reply:
Page delete.html supprimé